### PR TITLE
2023-07-14 Albert Larsan

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

**Comment by albertlarsan68** [comment]
My name is associated with not my twitter account

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Bootstrapping Rust with Albert Larsan"
+date: 2023-07-14T13:01:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-07-14-albert-larsan.mp3
+duration: "33:04"
+length: "31745254"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+
+Allen Wyma talks with [Albert Larsan](https://twitter.com/cernjai) about his work on bootstrapping the Rust compiler.
```
Comment (albertlarsan68) [inline _episodes/interviews/2023-07-14-albert-larsan.md:10]:
```suggestion
Allen Wyma talks with [Albert Larsan](https://github.com/albertlarsan68) about his work on bootstrapping the Rust compiler.
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@0:00] - Introduction to Bootstrap
+- [@1:57] - Building the standard library and compiler.
+- [@5:34] - The biggest challenges of bootstrapping RustC
```
Comment (albertlarsan68) [inline _episodes/interviews/2023-07-14-albert-larsan.md:25]:
```suggestion
- [@5:34] - The biggest challenges of bootstrapping Rustc
```
I think this is the correct case, I've never seen it written as RustC.

**Review by albertlarsan68 — CHANGES_REQUESTED** [review]
Thanks for having me!
A few corrections and I think we're good to publish!

**Review by albertlarsan68 — APPROVED** [review]
:ship: it!

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+title: "Bootstrapping Rust with Albert Larsan"
+date: 2023-07-14T13:01:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:3]:
```suggestion
date: 2023-07-14T14:08:00Z
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@0:00] - Introduction to Bootstrap
+- [@1:57] - Building the standard library and compiler.
+- [@5:34] - The biggest challenges of bootstrapping Rustc
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:25]:
```suggestion
- [@5:34] - The biggest challenges when bootstrapping `rustc` (the Rust compiler)
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+- [@11:26] - Why use Python to start the bootstrapping process?
+- [@13:08] - Running tests as part of the Rust API, ensuring that each component is thoroughly tested.
+- [@15:07] - Running tests on Linux, MacOS & Windows
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:28]:
```suggestion
- [@15:07] - Running tests on Linux, MacOS, and Windows
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@0:00] - Introduction to Bootstrap
+- [@1:57] - Building the standard library and compiler.
+- [@5:34] - The biggest challenges of bootstrapping Rustc
+- [@11:26] - Why use Python to start the bootstrapping process?
+- [@13:08] - Running tests as part of the Rust API, ensuring that each component is thoroughly tested.
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:27]:
```suggestion
- [@13:08] - Running tests as part of the Rust API to ensure that each component is well-tested.
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+- [@15:07] - Running tests on Linux, MacOS & Windows
+- [@18:54] - Features that the bootstrapping team for RustC has been working on.
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:29]:
```suggestion
- [@18:54] - Features that the `rustc` bootstrapping team has been working on.
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+- [@20:50] - Plans to run Clippy and fix issues.
+- [@23:41] - Stage redesign, verification, reproducibility, and different optimizations implemented in the compiler.
+- [@27:28] - Albert's advice on the process of bootstrapping Rustc.
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:32]:
```suggestion
- [@27:28] - Albert's advice on the process of bootstrapping Rust.
```

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+- [@31:05] - Importance of documentation for Rustc
```
Comment (jonhoo) [inline _episodes/interviews/2023-07-14-albert-larsan.md:33]:
```suggestion
- [@31:05] - The importance of documentation
```

**Review by jonhoo — APPROVED** [review]

File: _episodes/interviews/2023-07-14-albert-larsan.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+## Timestamps
+
+- [@0:00] - Introduction to Bootstrap
+- [@1:57] - Building the standard library and compiler.
+- [@5:34] - The biggest challenges when bootstrapping `rustc` (the Rust compiler)
+- [@11:26] - Why use Python to start the bootstrapping process?
+- [@13:08] - Running tests as part of the Rust API to ensure that each component is well-tested.
+- [@15:07] - Running tests on Linux, MacOS, and Windows
+- [@18:54] - Features that the `rustc` bootstrapping team has been working on.
```
Comment (albertlarsan68) [inline _episodes/interviews/2023-07-14-albert-larsan.md:29]:
```suggestion
- [@18:54] - Features that the Rust bootstrap team has been working on.
```

The name of the team is bootstrap, not bootstrapping

Comment (jonhoo) [reply _episodes/interviews/2023-07-14-albert-larsan.md]: Thanks!

**Pull request merged**